practices/simpleDesign/readme.md

```diff
@@ -0,0 +1,35 @@
+# Simple Design
+
+The simplest design is often (almost always) the best solution to a given problem. The simplest solution is the easiest to maintain and the easiest to change (assuming you have appropriate testing of course). The assertion here is that trying to "build for the future" or "build in extensibility" ends up being more costly as we cannot predict what future requirements will be. Solve for the now, iterate on future later.
+
+## The What and the Why
+
+### The What
+
+Simple design refers to designing your code in a simple, clean and elegant way, but not necessarily simplistic. The following are guidelines to simple design:
+
+1. "You Aren't Gonna Need It"(YAGNI): Design details should be implemented when you need them, not when you foresee you will need them.
+2. Avoid duplication: duplicate code is hard to read, understand and and maintain.
```

Comment:
should reference [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) here